uat/src/main/java/com.aws.greengrass/FileSteps.java

```diff
@@ -45,21 +48,22 @@ public FileSteps(Platform platform, TestContext testContext) {
     public void arrangeComponentLogFiles(int numFiles, String componentName) throws IOException {
         Path logsDirectory = testContext.installRoot().resolve("logs");
         LOGGER.info("Writing {} log files into {}", numFiles, logsDirectory.toString());
-
         if (!platform.files().exists(logsDirectory)) {
             throw new IllegalStateException("No logs directory");
         }
-
+        scenarioContext.put(componentName + "LogDirectory",logsDirectory.toString());
         if (componentName.equals("aws.greengrass.Nucleus")) {
+
             for (int i = 0; i < numFiles; i++) {
-                String fileName = String.format("greengrass_%d.log", i);
+                String fileName = String.format("greengrass_%s.log", i);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: i (declared at line 57). This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 58. Consider formatting this data with &#39;%d&#39; instead.

Reply:
This is correct this should be a %d